fades/helpers.py

```diff
@@ -256,3 +257,24 @@ def check_pypi_exists(dependencies):
                 logger.error("%s doesn't exists in PyPI.", dependency)
                 return False
     return True
+
+
+def list_venvs(index_path):
+    """List all venvs from an index file path and print info to stdout."""
+    if os.path.isfile(index_path):
+        tmplt = ("\nVirtualenv uuid:\t{uid}\n\ttimestamp:\t{dat}\n\tfull path:\t{pat}\n"
+                 "\tdependencies:\t{pac}\n\tinterpreter:\t{pyv}\n\toptions:\t{opt}\n")
+        venv_info = ""
+        with open(index_path) as jotason:
+            for jotason_line in jotason:
+                v_dct_get = json.loads(jotason_line).get
+                venv_info += tmplt.format(
+                    uid=v_dct_get("metadata")["env_path"][18:],
+                    pat=v_dct_get("metadata")["env_path"],
+                    pac=v_dct_get("installed"),
+                    pyv=v_dct_get("interpreter"),
+                    opt=v_dct_get("options"),
+                    dat=datetime.fromtimestamp(v_dct_get("timestamp")).replace(
```

Comment:
You can use `{:%Y-%m-%d %H:%m}`  (or similar) in tmplt and here you only need to use `fromtimestamp`